lib/util.js

```diff
@@ -471,3 +472,22 @@ exports.deprecate = (obj, prop, value, oldPath, newPath) => {
     },
   });
 };
+
+
+/**
+ * Gets random IPv6 Address from a block
+ *
+ * @param {string} block
+ * @returns {string}
+ */
+exports.getRandomIPv6 = block => {
+  let subnet = parseInt(block.split('/')[1]);
+  block = block.split('/')[0];
+  if (!subnet || subnet > 128 || subnet < 24) throw new Error('Invalid IPv6 subnet');
+  // Other errors will be thrown by ip6 itself
+  let ip;
+  if (subnet < 64) block = ip6.randomSubnet(block, subnet, 64, 1, true)[0];
```

Comment:
not sure what this does

Reply:
yep
currently working on removing that ip6 dependencie and thought the same
picking a random address from a /48 subnet should result in the same as first picking a random /64 subnet and then a /128 addr

Reply:
not really, it should use different /64 each time, and /64 itself has trillion+ ips so getting ip from same /64 few times in a row would be too common on /48

Reply:
thats not how stochastics work

first rolling /64 and then /128 blocks results in the same variance (in both /64 and /128) as directly rolling /128

it's not like directly rolling /128 gives you "worse randomness" for your ipv6 network part

Reply:
i mean random /128 from /48 gives random /128 from it
getting random /64 from /48 and then /128 from the /64 is the more efficent way to stop 429

Reply:
i normally hate ppl who do sth like this but i really wanna proof my point 😅
so feel free to try for yourself
https://gist.github.com/TimeForANinja/8403d624306abad4dcafbed148ddbaf0
the standard deviation says it all

Reply:
So you are going to remove ip6 dependency? or do i update readme and its technically ready to merge?

Reply:
the ip6 replacement is already kinda done

i'd finish & merge this pr first thou with ip6 (the way it is - maybe remove that /64 blocks first part ^^)
that way i still got some time to add tests & clean up my code
and i hate poking around in other peoples pr's

Reply:
Okay, i'll update readme after i will wake up then 👍

Reply:
this is still not resolved right?

Reply:
Well you said you are not sure what this does

Reply:
I think this can be marked as resolved